src/org/zaproxy/zap/extension/alertFilters/ContextAlertFilterPanel.java

```diff
@@ -103,11 +103,9 @@ public AlertFiltersMultipleOptionsPanel(ExtensionAlertFilters extension, AlertFi
 			super(model);
 			this.extension = extension;
 
-			getTable().getColumnExt(0).setPreferredWidth(20);
-			getTable().getColumnExt(1).setPreferredWidth(50);
-			getTable().getColumnExt(2).setMaxWidth(30);
-			getTable().getColumnExt(3).setPreferredWidth(20);
+			getTable().getColumnExt(0).setMaxWidth(70);
```

Comment:
I'll merge the current changes now (to be included in the new release of the add-on.)
This can be fixed later, it's not a big issue.